tests/integ/scala/test_dataframe_reader_suite.py

```diff
@@ -497,19 +568,12 @@ def test_read_metadata_column_from_stage(session, file_format):
     res = df.collect()
     assert res[0]["FILENAME"] == filename
 
-    # test that column name with str works
-    reader = session.read.with_metadata("metadata$filename", "metadata$file_row_number")
-    df = get_df_from_reader_and_file_format(reader, file_format)
-    res = df.collect()
-    assert res[0]["METADATA$FILENAME"] == filename
-    assert res[0]["METADATA$FILE_ROW_NUMBER"] >= 0
-
 
 @pytest.mark.parametrize("mode", ["select", "copy"])
-def test_read_json_with_no_schema(session, mode):
+def test_read_format_json_with_no_schema(session, mode):
```

Comment:
we could parametrize this test with `.json()` and `.format("json").load()` so that we know both end points work.

Reply:
unresolving this conversation. This change removes tests for `.json()`. can they be re-added. Even though it is called by `.load()`, we still treat implementation as back box and add tests for interface.